ex03.ex

```diff
@@ -149,8 +181,17 @@ defmodule Ex03 do
   Think a little about a nice way to lay this code out.
   """
 
-  def won . . . "your code"
-
+  def won({ a, b, c, d, e, f, g, h, i}) do
```

Comment:
## -2

I know there's an element of subjectivity here, but I would find it difficult to verify by eye all the {a, e, i} cases.

I was thinking of something like this

~~~ elixir
  def won1({ a, a, a,  _, _, _,  _, _, _ }), do: a
  def won1({ _, _, _,  a, a, a,  _, _, _ }), do: a
  def won1({ _, _, _,  _, _, _,  a, a, a }), do: a
  def won1({ a, _, _,  a, _, _,  a, _, _ }), do: a
  def won1({ _, a, _,  _, a, _,  _, a, _ }), do: a
  def won1({ _, _, a,  _, _, a,  _, _, a }), do: a
  def won1({ a, _, _,  _, a, _,  _, _, a }), do: a
  def won1({ _, _, a,  _, a, _,  a, _, _ }), do: a
  def won1(_), do: false
~~~

or this

~~~ elixir
  def won({ a, a, a,
            _, _, _,
            _, _, _ }), do: a

  def won({ _, _, _,
            a, a, a,
            _, _, _ }), do: a

  def won({ _, _, _,
            _, _, _,
            a, a, a }), do: a

  def won({ a, _, _,
            a, _, _,
            a, _, _ }), do: a

  def won({ _, a, _,
            _, a, _,
            _, a, _ }), do: a

  def won({ _, _, a,
            _, _, a,
            _, _, a }), do: a

  def won({ a, _, _,
            _, a, _,
            _, _, a }), do: a

  def won({ _, _, a,
            _, a, _,
            a, _, _ }), do: a

  def won(_), do: false
~~~